runtime/src/main/java/org/corfudb/runtime/object/ISMRStream.java

```diff
@@ -38,8 +38,6 @@
 
     void seek(long globalAddress);
 
```

Comment:
That is a very good question. I went over the codebase of Corfu and found that there is only one invocation of seek and that moreover that invocation is not necessary. I think we could delete the seek API because it does not make sense in the context of in-place GC.